yesod-markdown.cabal

```diff
@@ -1,5 +1,5 @@
 name:                   yesod-markdown
-version:                0.9.4
+version:                0.9.5
```

Comment:
Can you revert this? I bump the version separately from any given PR; also this is an API change, so I'll probably go to `0.10.0`